### DISCUSSION TITLE
Documentation for KeystoneJS Icons

### DISCUSSION THREAD

**Discussion by noor-codes** [keystonejs/keystone#7287] [Feature requests]
### Documentation for KeystoneJS Icons

So I've been working with a custom field and found out there is an official keystone-icons library which is awesome. 
The problem is, there is no visual docs to go through or select any of the icons when needed. 

I bet these icons are mostly used internally for building the Admin-UI but if exposed to the community with documentation, that would be great. It also helps developers to customize the Admin-UI with respect to the Keystone Design 

I can prepare the docs if there is a certain guideline I can follow.

 [Link to NPM KeystoneJS Icons Package](https://www.npmjs.com/package/@keystone-ui/icons)

Thanks

**Comment by asgeo1** [comment]
Was looking for some documentation on the icons today. Just a simple page on the main documentation site, that prints all of the icons, and lets you search/filter them. Would be super handy
